tea.go

```diff
@@ -595,7 +595,7 @@ func (p *Program) Run() (Model, error) {
 	model, err := p.eventLoop(model, cmds)
 	killed := p.ctx.Err() != nil
 	if killed {
-		err = fmt.Errorf("%w: %s", ErrProgramKilled, p.ctx.Err())
+		err = fmt.Errorf("%w: %w", ErrProgramKilled, p.ctx.Err())
```

Comment:
Wow, TIL you cannot double-wrap

Reply:
Yeah, it does something similar to `errors.Join` when you double wrap errors. This *does* break `errors.Unwrap`, since that function only works on singly-wrapped errors, but `errors.Is` and `errors.As` continues to work as expected.